README.md

```diff
@@ -9,26 +9,14 @@ _Installs dependencies from a local checkout, and keeps them in sync, without th
 
 # Summary
 
-Relative deps introduces an additional dependency section in `package.json`, called `relativeDependencies`.
-This section contains paths to the local sources of any dependency, that will be built and installed over the publicly available versions, when needed.
+Relative deps introduces an additional dependency section in a config file called `relative-deps.json`
```

Comment:
Personally, I hate JSON. I mean, you can't even add comments to it. Many popular node packages use `cosmiconfig` to resolve configuration files in a variety of formats: JSON, JS, YAML. Should we offer the same?

Reply:
Oh cool! I'll take a deeper look at `cosmiconfig`. Seems like a great solution! Thanks